File: 백준/Silver/11723. 집합/집합.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.Arrays;
import java.util.StringTokenizer;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringBuilder sb = new StringBuilder();
        int M = Integer.parseInt(br.readLine());
        boolean[] S = new boolean[21];

        while (M -- > 0) {
            StringTokenizer st = new StringTokenizer(br.readLine());
            String command = st.nextToken();
            int x = 0;

            if (!command.equals("all") && !command.equals("empty")) {
                x = Integer.parseInt(st.nextToken());
            }
            switch (command) {
                case "add":
                    S[x] = true;
                    break;
                case "remove":
                    S[x] = false;
                    break;
                case "check":
                    if (S[x]) {
                        sb.append("1").append("\n");
                    } else {
                        sb.append("0").append("\n");
                    }
                    break;
                case "toggle":
                    if (S[x]) { S[x] = false; } else { S[x] = true; }
                    break;
                case "all":
                    Arrays.fill(S, true);
                    break;
                case "empty":
                    Arrays.fill(S, false);
                    break;
            }
        }

        System.out.println(sb);
    }
}

File: 백준/Silver/11723. 집합/집합.py
import sys

inputValue = sys.stdin.readline
outputWrite = sys.stdout.write

m = int(inputValue())
arr = [False] * 21

for _ in range(m):
    line = inputValue().split()

    if line[0] != 'all' and line[0] != 'empty':
        x = int(line[1])
        if line[0] == 'add':
            arr[x] = True
        elif line[0] == 'remove':
            arr[x] = False
        elif line[0] == 'check':
            outputWrite('1\n' if arr[x] else '0\n')
        elif line[0] == 'toggle':
            arr[x] = not arr[x]
    elif line[0] == 'all':
        for i in range(1, 21):
            arr[i] = True
    elif line[0] == 'empty':
        for i in range(1, 21):
            arr[i] = False